onnxscript/backend/onnx_export.py

```diff
@@ -26,17 +27,17 @@
 {% for domain, version in unique_function_domain_version: %}
 {{ domain }}{{ version }} = Opset("{{ domain }}", {{ version }}){% endfor %}
 {% for domain, name, fct in functions: %}
-@script({{ domain }}{{ version }})
-def {{ python_make_node_name(fct['proto'].domain, 1, fct['proto'].name) }}({{
-    ", ".join(map(rename, fct['proto'].input)) }}):
-    # attributes are missing
+@script({{ domain }}1)
```

Comment:
Was the 1 intended?

Reply:
Yes, that's what the rest of the code-generator uses.